### ISSUE TITLE
APIv2 test: test_copy_to_container: False is not true

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#10948]
I'm seeing this flake on almost every PR:
```
[+0215s] ======================================================================
[+0215s] FAIL: test_copy_to_container (compat.test_containers.TestContainers)
[+0215s] ----------------------------------------------------------------------
[+0215s] Traceback (most recent call last):
[+0215s]   File "/var/tmp/go/src/github.com/containers/podman/test/python/docker/compat/test_containers.py", line 232, in test_copy_to_container
[+0215s]     self.assertTrue(out.startswith(b'1042:1043'), "assert correct uid/gid")
[+0215s] AssertionError: False is not true : assert correct uid/gid
[+0215s] 
[+0215s] ----------------------------------------------------------------------
```
For example: [here](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/5246553477611520/html/apiv2-podman-fedora-34-root-host.log.html) 

When fixed, could you please make the error message a little bit more helpful than "false is not true"?

**Comment by edsantiago** [comment]
@matejvasek PTAL

**Comment by matejvasek** [comment]
@edsantiago I'll look into it, my guess is that there is something before expected string "1042:1043". Actually the output should only contain "1042:1043" and nothing else, but when running in CI there was some superfluous output so I changed exact match to `startswith`. If it's the case changing it to `contains` would help.

**Comment by edsantiago** [comment]
Here is [useful data](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/5795692124831744/html/apiv2-podman-fedora-34-root-host.log.html). The failure is:
```
[+0218s] ======================================================================
[+0218s] FAIL: test_copy_to_container (compat.test_containers.TestContainers)
[+0218s] ----------------------------------------------------------------------
[+0218s] Traceback (most recent call last):
[+0218s]   File "/var/tmp/go/src/github.com/containers/podman/test/python/docker/compat/test_containers.py", line 232, in test_copy_to_container
[+0218s]     self.assertEqual(out.rstrip(), b'1042:1043')
[+0218s] AssertionError: b'1042:1043\n[conmon:d]: exec with attach is waiting[86 chars]rent' != b'1042:1043'
[+0218s] 
[+0218s] ======================================================================
[+0218s] FAIL: test_mount_preexisting_dir (compat.test_containers.TestContainers)
[+0218s] ----------------------------------------------------------------------
[+0218s] Traceback (most recent call last):
[+0218s]   File "/var/tmp/go/src/github.com/containers/podman/test/python/docker/compat/test_containers.py", line 253, in test_mount_preexisting_dir
[+0218s]     self.assertEqual(out.rstrip(), b'1042:1043')
[+0218s] AssertionError: b'1042:1043\n[conmon:d]: exec with attach is waiting[86 chars]rent' != b'1042:1043'
[+0218s] 
```
In particular, there's an unexpected and unwanted "`[conmon:d]: exec blah blah". The root cause seems to be https://github.com/containers/podman/blob/main/test/python/docker/__init__.py#L30-L32  -- in particular, it looks like `$DEBUG` is set somewhere.

Soapbox: `$DEBUG` is an inappropriate name for a variable. Debug variables should have clear, unambiguous, specific names that cannot be accidentally set. I will fix this in my PR.

**Comment by matejvasek** [comment]
@edsantiago  This is really sneaky one, I run CI action three time as it always passed, not I am trying to reproduce it locally with `DEBUG=true` and still no success.

**Comment by matejvasek** [comment]
I modified my test locally to use regex so it didn't mind superfluous output.

**Comment by edsantiago** [comment]
@matejvasek I have a fix in #10950
